cli/sync_service/README.md

```diff
@@ -0,0 +1,362 @@
+    
+result of the command:
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "policy-basicres.tgz",
+        "objectType": "model"
+      },
+      {
+        "objectID": "policy-multires.tgz",
+        "objectType": "model"
+      },
+      {
+        "objectID": "lily1",
+        "objectType": "model"
+      }
+    ]
+    
+We can specify objectType and objectID to check the object we just published
+
+    hzn mms object list --objectType=model --objectId=lily1
+    
+result of the command:
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list --objectType=model --objectId=lily1
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "lily1",
+        "objectType": "model"
+      }
+    ]
+    
+To show the full information of MMS object metadata, you can add `-l` to the command
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list --objectType=model --objectId=lily1 -l
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "lily1",
+        "objectType": "model",
+        "destinationOrgID": "userdev",
+        "destinationID": "",
+        "destinationType": "",
+        "destinationsList": [],
+        "destinationPolicy": {
+          "properties": [],
+          "constraints": [],
+          "services": [
+            {
+              "orgID": "e2edev@somecomp.com",
+              "arch": "amd64",
+              "serviceName": "my.company.com.services.usehello2",
+              "version": "1.0.0"
+            }
+          ],
+          "timestamp": 1578594025910476311
+        },
+        "expiration": "",
+        "version": "0.0.1",
+        "description": "test policy",
+        "link": "",
+        "inactive": false,
+        "activationTime": "",
+        "noData": false,
+        "metaOnly": false,
+        "destinationDataUri": "",
+        "sourceDataUri": "",
+        "consumers": 1,
+        "autodelete": false,
+        "originID": "Cloud",
+        "originType": "Cloud",
+        "deleted": false,
+        "instanceID": 1578594025929,
+        "dataID": 1578594025929,
+        "objectSize": 6,
+        "chunkSize": 122880
+      }
+    ]
+
+
+
+If you want to show object status and destinations along with the object, you can add `-d` to the command:
+
+    root@lily-test:~/go/src/github.com/open-horizon/anax# hzn mms object list --objectType=model --objectId=lily1 -d
+    Listing objects in org userdev:
+    [
+      {
+        "objectID": "lily1",
+        "objectType": "model",
+        "destinations": [
+          {
+            "destinationType": "openhorizon.edgenode",
+            "destinationID": "an12345",
+            "status": "delivered",
+            "message": ""
+          }
+        ],
+        "objectStatus": "ready"
+      }
+    ]
+    
+Full list of flags can be obtained by `hzn mms object list --help`
+
```

Comment:
Dont show the help output, it will get stale. But you could talk about some of the more advanced filtering options which havent been covered.